bundle/metadata/dependencies.yaml

```diff
@@ -3,3 +3,7 @@ dependencies:
   value:
     packageName: ocs-operator
     version: ">=4.8.0 <4.9.0 || 4.9.0"
+- type: olm.package
```

Comment:
We are deciding to manage it via the odf-operator fully and won't rely on the OLM to resolve it. It can be removed.